test/inductor/test_torchinductor.py

```diff
@@ -7095,6 +7095,27 @@ def fn(tensor, index, source):
             ),
         )
 
+    def test_torch_dynamo_codegen_pow(self):
+        def pow(x):
+            return x**2
+
+        x = torch.randn(8, device=self.device)
```

Comment:
We want to test NumPy here not PyTorch!

Reply:
Put it in  `test_misc.py` better, as we don't want to test it for CUDA. There are a few other numpy tests there as well

Reply:
My bad, brain fart moment right there!